guides/common/modules/proc_configuring-the-freeipa-server-to-use-cross-forest-trust.adoc

```diff
@@ -14,9 +14,17 @@ For example:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-[nss]
-user_attributes=+mail, +sn, +givenname
+[domain/lab.example.com]
+...
+krb5_store_password_if_offline = True
+ldap_user_extra_attrs=email:mail, lastname:sn, firstname:givenname
 
-[domain/EXAMPLE]
-ldap_user_extra_attrs=mail, sn, givenname
+[ifp]
```

Comment:
do we need the `ifp` section/service for anything (besides the `dbus-send` call below that we added now?)

Reply:
@waldirio Can you help us to clarify?

Reply:
Hello,

yes, there is an entry that should be added, as below
```
[ifp]
allowed_uids = ipaapi, root
user_attributes=+email, +firstname, +lastname
```

This will be necessary during this process of satellite/IDM authentication + cross forest (AD).


Please, let me know if you have additional questions or concerns.
Waldirio